sources/Renderer/Metal/MTCore.mm

```diff
@@ -23,8 +24,8 @@ void MTThrowIfFailed(NSError* error, const char* info)
         
         NSString* errorMsg = [error localizedDescription];
         s += [errorMsg cStringUsingEncoding:NSUTF8StringEncoding];
-        
-        throw std::runtime_error(s);
+
+        LLGL_TRAP(s.c_str());
```

Comment:
This is a pitfall because `LLGL_TRAP()` expects a format string just like `printf()`, so unless you can guarantee that the error message from `NSError` never contains the `%` character this can crash because no corresponding argument is provided in the variadic argument list. So this needs to be `LLGL_TRAP("%s", s.c_str())`.